### DISCUSSION TITLE
最近發票開始兌獎了，大家有中嗎

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2279] [General]
我從以前到現在還沒有中過一次，唯一最接近的一次是號碼全中除了最後一位，有沒有人可以提供中獎方法

**Comment by Mminniehuang** [comment]
我之前是有聽說載具的中獎機率比傳統紙本發票的中獎機率高，所以現在都用載具，而且因為爸媽懶得申請，所以他們也直接讓我傳載具條碼的截圖給他們用。發票數量變多之後，前兩期都有幸運的中一兩百塊，雖然不多但看了很開心😆

**Comment by Elaine4488** [comment]
看到大家用雲端發票都有中獎，但我到現在都還沒中過一次。

**Comment by christaaah** [comment]
我從沒有中過一次，就算使用雲端發票，還是連２００塊都沒有，甚至雲端發票還有額外獎勵，但是看到有人發圖說中獎名單感覺連續中了很多次，還是金額大的那種，都覺得不可思議。

**Comment by kzx-creat** [comment]
我差不多是三四年前開始都刷載具，因為聽說電子載具中獎機比較高?體感是如此沒錯。前幾個月有中過500跟200的，但這次兌獎沒中QQ

**Comment by yue083139** [comment]
我覺得雲端發票的好處在於自動對獎，之前不知道有載具可以用的時候都是累積了一堆的發票拿回家給爸媽對，或是乾脆直接丟了，現在用雲端發票或多或少有一點中獎的機會了，像我這期就第一次中獎，雖然只有200塊，但畢竟是意外之財所以還是蠻開心的，希望下次還能繼續中獎。

**Comment by Mminniehuang** [comment]
雲端發票的方便性和環保我覺得是很強大的優點，但也讓我想到我的家人最開始不願意使用的原因就是，對他們來說每期把厚厚一疊發票拿出來對是很大的樂趣，那種緊張又期待的感覺，或是自己發現中獎驚喜感和喜悅是被雲端發票剝奪的！
但就是要記得收好還有兌獎，或許其實不少人都因此跟獎金擦肩而過

**Comment by 113607514** [comment]
我從來沒中過一次，唯一一次最接近的時候是號碼全中，結果最後一位差了一個，真的是差點就破天荒了！不知道有沒有人能提供一個「中獎公式」，讓我終於能笑到最後，哈哈！

**Comment by ed112601513** [comment]
平時幾乎都用紙本發票，只有像電信繳費之類的少數情況才會把發票存在雲端，結果我對了幾百張紙本發票連200元都中不了，然後雲端發票中500，雖然很喜歡紙本兌獎的驚喜感，但是雲端中獎的機率感覺比較高

**Comment by qiu0109** [comment]
雖然都說雲端發票中獎機率比較高，但是我從用到現在也只中過一次200而已，感覺也沒有那麼容易中獎。之前有在網路上看到有人雲端發票一個季度累積中了一萬多，真希望他分一點運氣給我。

**Comment by kai1660** [comment]
沒有中，一張都沒有，我使用傳統發票，我下一次要改為使用載具，因為載具有而外加開特別獎，可以增加中獎機會，不但可以節能減碳環保救地球，還可以有而外的獎金，希望大家都可以愛護地球。

**Comment by vu45tp6** [comment]
每次都要看到那個沒有中的標語和大家都有中獎的限動截圖

**Comment by RoyLin1014** [comment]
雲端發票比較好中，紙本好久沒中獎了

**Comment by Twicky123** [comment]
最近開始使用載具，因為紙本發票整理起來太麻煩了，用載具可以讓app自動幫你兌獎，因此我推薦大家都來用!

**Comment by lilipop6405** [comment]
我對紙本的只有對過200一次而已，話說我從來沒有使用過載具，跟老人一樣 = =。

**Comment by SendaiHatsuki** [comment]
已經好幾期沒有中過了，而且以前也只拿過200快而已，其他大一點的獎都沒拿過，這跟我基本無緣。中央出的那一個一千萬的真的很羨慕。

**Comment by mattwang1222** [comment]
這種東西是機率問題，所以大家的回答會有所不同，通常接近隨機分布

**Comment by Kaz9487** [comment]
沒有，這期沒中，沒有額外的加餐金了。中央附近有提供載具或發票的店家不多，不然如果平常吃飯也都有發票載具的話，應該也能提高中獎次數

**Comment by hollowha** [comment]
想要利用發票來分析財務狀況，幫助你理財嗎? 試試發票醫生!! 我們是中央大學的學生團隊
https://ssp.moe.gov.tw/cases/1349

**Comment by jaychang1027** [comment]
再過幾天就要開獎了，希望這次老天會眷顧我，我已經大概一年沒中過了，我都用載具，比一般發票多了一次中獎的機會，而且我的發票數都不少，但還是每次都槓龜，真的有夠難受

**Comment by yuku-huang** [comment]
發票中獎真的是運氣問題，沒什麼技巧，但我朋友有個習慣是每次都存雲端發票，然後參加加碼活動，像是財政部那種加碼200萬或是特定商店再抽獎，反而比較容易中。還有就是發票存好別忘了對，很多人根本錯過領獎時間就白費了。運氣這種事，撐久總會輪到你一次啦。

**Comment by shrshu38** [comment]
這次發票我中了五百，對獎當下超驚喜，平常根本沒抱期待。雖然不多，但有種被生活獎勵的感覺，剛好是那次去全聯買菜的價錢！

**Comment by ryan8967** [comment]
3年前中過人生的第一張電子發票500元，但直到現還沒有再中過。可能是因為都待在學校，大部分消費都是沒有發票的。有發票的消費只有全家和全聯而已。

**Comment by brianou583** [comment]
現在政策為了減少紙本發票增加載具發票，新增雲端發票專屬獎，而且中獎機率還算蠻到的我大約每兩次開獎會中一次。如果想多中獎可以多使用載具！

**Comment by 12388833** [comment]
雖然大獎機率很低，但中個兩百也會讓人開心一整天。不管有沒有中，記得定期對獎、把發票存載具，除了有機會中獎，也能參加加碼活動，還對環保有幫助。我也是最近才開始使用載具，發現真的蠻方便的，錢包不會滿滿的都是發票，不過這次沒有中獎，希望下次有機會中。

**Comment by yunnnnchieh** [comment]
雖然沒有中獎，但有載具之後都習慣用雲端發票，真的方便很多，錢包也不會很多張紙。載具統一管理，不怕發票遺失，也不用自己一張一張對，系統中獎會直接通知，還能自己匯款到銀行帳戶，不只是環保，也比較不會錯過中獎機會。

**Comment by ichenx2** [comment]
昨天又開獎了，我有107張發票還是一張都沒中，只能說這真的很看運氣⋯會中就是會中，像我就是都不會中。但也蠻推薦大家用載具！我之前中過的幾次都是中雲端發票專屬獎，而且一中就至少500。像我出門習慣只帶手機，如果消費刷載具也不用拿一對單據回來，對我來說也是一種方便！
